internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -102,6 +102,10 @@ func (rp *RegistryPod) init(cfg *operator.Configuration) error {
 
 	// TODO(rashmi/venkat): If FBCPath is empty, assign it a default value (maybe something thats hosted on a registry)
 
+	if rp.FBCdir == "" {
```

Comment:
Let's move this check to `rp.validate()` as that function checks for RegistryPod required fields to be set and throws an error if not.